### DISCUSSION TITLE
How to use FeatureStore and GraphStore

### DISCUSSION THREAD

**Discussion by KevinCrp** [pyg-team/pytorch_geometric#7328] [Q&A]
Hi,

I'm trying to use FeatureStore and GraphStore in my project, but when I test the given example in the doc, it's fails.
Someone can provide a full example, please :)

**Comment by rusty1s** [comment]
https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/testing/graph_store.py and https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/testing/feature_store.py show some simple example implementations of `FeatureStore` and `GraphStore`. Hopefully this gets you going. Let me know if you have any more questions.

**Reply by kayleigh-li-imprivata** [reply]
Hi @rusty1s I know there's Kuzu support on graphstore and featurestore and I found [one example](https://colab.research.google.com/drive/12fOSqPm1HQTz_m9caRW7E_92vaeD9xq6#scrollTo=fmXWs1dKIzD8) for node prediction, in which the `NeighborLoader` does not work on our data (kernal got killed immediately when calling it). Do you know if link loader is supported as well (hoping to use `LinkNeighborLoader`)? If so, need the kuzu.GraphStore be converted to pyg's GraphStore first, or edge_index needs to be pre-generated and stored in kuzu somehow in order for linkneighborloader to work (our data is heterogenous)?

Any help would be greatly appreciated.

![image](https://github.com/user-attachments/assets/ea0a11a2-181d-4a19-a6b5-b16ba0ca9a8e)
